test/basic.html

```diff
@@ -22,6 +22,17 @@
   <link rel="import" href="../paper-radio-button.html">
 
 </head>
+<style is="custom-style">
+  paper-radio-button.large {
+    --paper-radio-button-size: 48px;
+  }
+  paper-radio-button.medium {
+    --paper-radio-button-size: 24px;
+  }
+  paper-radio-button.small {
+    --paper-radio-button-size: 12px;
+  }
+</style>
```

Comment:
Though, we won't be able to get rid of the margin between the label and button without the `:empty` pseudo-selector I think. :/